### DISCUSSION TITLE
Filtering by checkpoint or lora type (exclude Anime)

### DISCUSSION THREAD

**Discussion by Aamir3d** [civitai/civitai#276] [Ideas]
Of late, the Lora section is flooded by anime uploads that all look very similar.
I have no problems with uploads, but filtering or finding models that I need to update or use (retro, photorealistic etc.) is very difficult.
Could we have a way to filter out unwanted tags eg. Anime, Hentai etc so that the UI is less cluttered?

**Comment by mozimi** [comment]
I know that this may be a big ask, but it would be cool too if in the titles - model makers could specifically mention style vs character vs workaround (or something to that effect - charturner comes to mind) vs object vs outfit. Lots of times I'm looking and I don't understand if they trained the character from a show (I don't know any anime shows) or if they are training the style.
